backends/hip-shared/ceed-hip-shared-basis.c

```diff
@@ -704,10 +713,15 @@ __device__ void weight3d(const CeedInt nelem, const CeedScalar *qweight1d,
 //------------------------------------------------------------------------------
 extern "C" __launch_bounds__(INTERP_BLKSIZE) __global__ void interp(
                                   const CeedInt nelem, const int transpose,
-                                  const CeedScalar *c_B,
+                                  CeedScalar *d_interp1d,
                                   const CeedScalar *__restrict__ d_U,
                                   CeedScalar *__restrict__ d_V) {
+
   HIP_DYNAMIC_SHARED( double, slice)
+  // load interp1d into shared memory
+  __shared__ double c_B[P1D*Q1D];
```

Comment:
I would call it `s_B`, the semantic I tried to use is: `s_*` for shared memory, `r_*` for registers, `d_*` for device memory, and `c_*` for constant memory.

Reply:
Good catch, I fixed this in 4e6ce5a03b715256484190a896dd2a02cda2ad8b